notebooks/validation/hydro_generation_validation.ipynb

```diff
@@ -150,26 +150,46 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": 14,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "18.68199810116899"
+       "26428.447600321193"
       ]
      },
-     "execution_count": 9,
+     "execution_count": 14,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
     "ror_gen_index = n.generators[n.generators.carrier == \"ror\"].index\n",
-    "hydroenergy_by_ror = n.generators_t.p[ror_gen_index].sum().sum() / 1000000\n",
+    "hydroenergy_by_ror = n.generators_t.p[ror_gen_index].mean().sum() / 1000 * 8760\n",
```

Comment:
This is now GWh instead of TWh or not?

Reply:
The above is to avoid problems when snapshots do not contain 8760h.
In the case the snapshots do not contain such number of units, the first sum is not the total yearly production by plant, but a fraction. By using mean, we compute the hourly average whatever the length of the snapshot
Hence to compute the yearly total we can multiply such number by 8760.

Reply:
Done